### PR TITLE
Update setup-dev.md file

### PR DESCRIPTION
**Pull request by IrushiL**

### CODE REVIEW

File: docs/content/src/develop/setup-dev.md [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
+* A Text Editor or an IDE
+> **Tip**: For a better development experience, download and install [VS Code](https://code.visualstudio.com/Download) from the Visual Studio website
+if you do not have it already and install the [Ballerina Integrator Extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina-integrator) in VS Code Marketplace.
 
-## Setting up IDEs
+![alt text](../assets/img/vs-code-plugin.png)
 
-You can use your [favorite editor to work on Ballerina code](https://ballerina.io/learn/tools-ides/), however, we recommend you use VS Code as we have made some improvements to it for use in integration scenarios.
+Once installed, the following `Ballerina Integrator Dashboard` shown below could be used for creating new Ballerina
+projects, creating modules using available module templates and browsing the available module templates. The sequence
```
Comment (samgnaniah) [inline docs/content/src/develop/setup-dev.md]:
```suggestion
projects, creating modules using available module templates, and browsing the available module templates. The sequence
```

File: docs/content/src/develop/setup-dev.md [hunk 2]
```diff
@@ -1,20 +1,44 @@
...
+
+### Installing on Linux
+
+Download the DEB file, use the following command to install the Ballerina Integrator. Replace `<ballerina-home>/<ballerina-binary>.deb` with the actual file path. This installs the Ballerina Integrator in the `/usr/lib/ballerina` directory.
+```
+dpkg -i <ballerina-binary>.deb
+```
+
+### What's Next
+Once you have successfully installed Ballerina, try out the [Quick Start Guide](../get-started/quick-start-guide.md) and take Ballerina Integrator for its first spin.
```
Comment (samgnaniah) [inline docs/content/src/develop/setup-dev.md]: Did you test this link using mkdocs serve?

Comment (IrushiL) [reply docs/content/src/develop/setup-dev.md]: Yes, it works.

File: docs/content/src/develop/setup-dev.md (hunk 1 above)
Comment (IrushiL) [reply docs/content/src/develop/setup-dev.md]: Fixed

**Pull request merged**